src/components/utilities/languages/French.json

```diff
@@ -62,19 +63,23 @@
   "via": "via",
   "ECONOMY": "ÉCONOMIQUE",
   "BUSINESS": "BUSINESS",
-  "FIRST CLASS": "PREMIÈRE CLASSE",  
+  "FIRST CLASS": "PREMIÈRE CLASSE",
   "FLIGHT": "VOL",
   "DISTANCE": "DISTANCE",
   "AIRPLANE": "AVION",
   "CLASS": "CLASSE",
-  "CALCULATE CARBON FOOTPRINT": "CALCULER L'EMPREINTE CARBONE", 
+  "CALCULATE CARBON FOOTPRINT": "CALCULER L'EMPREINTE CARBONE",
 
   "METRIC TONS": "TONNES MÉTRIQUES",
+  "tons of CO2": "tonnes de CO2",
+  "Emissions Statement": "les émissions de CO2 de votre vol seront neutralisées en 1 an",
```

Comment:
same key change (we're also going to need non google translate translations in the future)